dist/scss/Components/FormGroup/style.scss

```diff
@@ -0,0 +1,46 @@
+@import './../../vars';
+
+// Normal State
+$label-Dark-Text: #4729b3;
+$input-field-BG: #fff;
+$input-field-border: 1px solid #4729b3;
+$placeholder-text: #555;
+$description-text: #555;
+
+// Hover
+$input-field-border-hover: 1px solid #4729b3;
+$transition-duration-super-short: 0.15s;
+$transition: border-color 0.15s ease-in-out;
+
+// Active
+$label-Dark-Text-active: $primary-color;
+$input-field-border-active: 1px solid $primary-color;
+$box-shadow: 0 0 6px 3px $primary-color;
+$transition-active: border-color $transition-duration-super-short ease-in-out,
+    box-shadow $transition-duration-super-short ease-in-out;
+
+.form-group-of-trust {
+    padding: $MARGIN 0;
+    margin: $SIZE;
+
+    .form-label-of-trust {
+        color: $label-Dark-Text;
+    }
+
+    .form-input-of-trust {
+        border: $input-field-border;
+        color: $placeholder-text;
+
+        &:hover {
+            border: $input-field-border-hover;
+            transition: $transition;
+        }
+
+        &:active {
+            border: $input-field-border-active;
+            box-shadow: $box-shadow;
+            color: $label-Dark-Text-active;
```

Comment:
`$label-Dark-Text` and `$label-Dark-Text-active`  - all small letters and in all other variables where it is not. You can also find scss-lint rule for it, so it would trigger an error next time.